js/app.js

```diff
@@ -1,35 +1,17 @@
 /* Set the width of the side navigation to 250px and the left margin of the page content to 250px and add a black background color to body */
 function openNav() {
   $u('#mySidenav')
-    .removeClass('closedsidenav')
     .addClass('opensidenav');
```

Comment:
.addClass() add the classname to the classlist for the target node.
```javascript
  var addClass = function(className) {
    target.classList.add(className);
    return this;
  };
```
But now we're not removing 'closedsidenav' class from the classlist for the target node. So the target node still has that class?? If it has two classes, both clsoedsidenav and opensidenav, how does the CSS know which class to render?